src/nvim/ui.c

```diff
@@ -166,6 +166,88 @@ void ui_event(char *name, Array args)
   }
 }
 
+/// @param[in] aep data to convert
+/// @param[out] out structure that will be sent to UIs
+HlAttrs attrentry2hlattrs(const attrentry_T *aep, bool use_rgb)
+{
+  assert(aep);
+
+  HlAttrs attrs = HLATTRS_INIT;
+  int mask = 0;
+
+  mask = use_rgb ? aep->rgb_ae_attr : aep->cterm_ae_attr;
+
+  attrs.bold = mask & HL_BOLD;
+  attrs.underline = mask & HL_UNDERLINE;
+  attrs.undercurl = mask & HL_UNDERCURL;
+  attrs.italic = mask & HL_ITALIC;
+  attrs.reverse = mask & (HL_INVERSE | HL_STANDOUT);
+
+  if (use_rgb) {
+    if (aep->rgb_fg_color != normal_fg) {
```

Comment:
This kind of calls make the export fail for the "Normal" highlight. I'll try to replace "normal_fg" by -1 and see if htis has any adverse impact.